### PR TITLE
fix: adding table for admin-sales-by-event

### LINKED ISSUE #5163: Admin: Sales Dashboard Not Showing Events that are Selling Tickets
The Admin Sales dashboards are not showing events that are selling tickets and which have generated sales.

Expected: 
* All events should actually show up. 
* Events that have not made a sale should show up with 0, but events that have made sales should show up accordingly.
* Latest events (with a newer date) should be on top of the list.
* The columns should be sortable where possible

Compare: https://eventyay.com/admin/sales

Screenshot from 2020-09-27 20-36-20

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes #5163 
Screenshot from 2020-10-05 21-04-39

Screenshot from 2020-10-05 21-09-42

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Please group the status sales like in previous UI and use standard date cell like in invoice table

**Comment by iamareebjamal** [comment]
Currency and correct date format is missing as well

Previous UI:
image

**Comment by maze-runnar** [comment]
scrnli_10_6_2020_12-20-24 AM

File: app/templates/components/ui-table/cell/admin/sales/cell-amount.hbs [hunk 1]
```diff
@@ -0,0 +1 @@
+{{currency-symbol this.extraRecords.paymentCurrency}} {{format-money this.record}}
```
Comment (iamareebjamal) [inline app/templates/components/ui-table/cell/admin/sales/cell-amount.hbs:1]:
```suggestion
{{currency-symbol @extraRecords.paymentCurrency}} {{format-money @record}}
```

**Comment by iamareebjamal** [comment]
> Please group the status sales like in previous UI and use standard date cell like in invoice table

Date is still in wrong format. Please check all points before finalizing

File: app/controllers/admin/sales/index.js [hunk 2]
```diff
@@ -1,6 +1,79 @@
...
+      valuePath       : 'name',
+      isSortable      : true,
+      headerComponent : 'tables/headers/sort'
+    },
+    {
+      name            : 'Event Date',
+      valuePath       : 'startsAt',
+      isSortable      : true,
+      headerComponent : 'tables/headers/sort',
+      cellComponent   : 'ui-table/cell/cell-simple-date'
```
Comment (iamareebjamal) [inline app/controllers/admin/sales/index.js:23]:
```suggestion
      cellComponent : 'ui-table/cell/cell-date',
      options       : {
        timezone   : 'UTC',
        dateFormat : 'MMMM DD, YYYY'
      }
```

**Comment by iamareebjamal** [comment]
Amount column doesn't show anything, just 0 always. And build is also failing

**Comment by iamareebjamal** [comment]
Please see last commit. Many changes were unnecessary

**Pull request merged**